Jenkinsfile_pipeline_test

```diff
@@ -50,6 +50,8 @@ node {
   } catch (err) {
     notifyBuildFailure channel: channel
     throw err
+  } finally {
+    deleteDir()
```

Comment:
Should we remove it from checkout step ? Or may be just let it be there as there is no harm?

Reply:
I think always good to have it during checkout. Forget this.